meta-genivi-dev/meta-ivi/recipes-extended/node-state-manager/node-state-manager_2.0.0.bbappend

```diff
@@ -1,5 +1,13 @@
-EXTRA_OECONF_append=" --with-nsmc=NodeStateMachineTest "
+NSMC="SimpleNodeStateMachine"
+EXTRA_OECONF_append=" --with-nsmc=${NSMC} "
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-do_install_append() {
-    install -m 755 ${B}/NodeStateMachineTest/NodeStateTest ${D}${bindir}/NodeStateTest
+SRC_URI_append = "file://0001-Build-SimpleNodeStateMachine.patch "
+
+do_configure_prepend() {
+    git clone https://github.com/GENIVI/simple-node-state-machine.git ${WORKDIR}/git/${NSMC}
```

Comment:
+1 although I don't think we have hard requirements to only use SHAs to allow non-connected builds it would be good to keep as much download as possible in the fetch task.